app/assets/stylesheets/frontpage.css.scss

```diff
@@ -0,0 +1,111 @@
+.newest-shows
+{
+  width: 30%;
+  background: {
+    color: #170563;
```

Comment:
Color literals like `#170563` should only be used in variable declarations; they should be referred to via variable everywhere else.